### ISSUE TITLE
4.0beta17/4.0rc1 Project stopped working in exported version after upgrade

### ISSUE THREAD

**Issue by patrykjadamczyk** [godotengine/godot#73267]
### Godot version

v4.0.rc1.mono.official [8843d9ad3]

### System information

Windows 10, Forward+ (Vulcan), NVIDIA GeForce GTX 1050 Ti

### Issue description

I made simple project with panorama sky and basic ui with 3d scene in background. In beta 16 (v4.0.beta16.mono.official [518b9e580]) project works both in editor using play and using export to windows.
In beta 17 (v4.0.beta17.mono.official [c40020513]) I noticed that my project works normally when using play in editor but when i export project for windows. Compiled build doesn't work and makes this output:
```
ERROR: .NET: Failed to load hostfxr
   at: (modules/mono/mono_gd/gd_mono.cpp:394)
ERROR: Unable to open file: res://.godot/imported/industrial_sunset_02_puresky_4k.exr-fb33adb88c5253d1c44585a4ba1f1a66.bptc.ctex.
   at: (scene/resources/texture.cpp:846)
ERROR: Failed loading resource: res://.godot/imported/industrial_sunset_02_puresky_4k.exr-fb33adb88c5253d1c44585a4ba1f1a66.bptc.ctex. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:221)
ERROR: Failed loading resource: res://industrial_sunset_02_puresky_4k.exr. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:221)
ERROR: Can't load dependency: res://industrial_sunset_02_puresky_4k.exr.
   at: (core/io/resource_format_binary.cpp:696)
ERROR: Failed loading resource: res://.godot/exported/133200997/export-83e0d428f1cad8b2c6d76f22094daa71-Env.res. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:221)
ERROR: Can't load dependency: res://Env.tres.
   at: (core/io/resource_format_binary.cpp:696)
ERROR: Failed loading resource: res://.godot/exported/133200997/export-bcb0d2eb5949c52b6a65bfe9de3e985b-Main.scn. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:221)
ERROR: Failed loading scene: res://Main.tscn
   at: (main/main.cpp:2927)
ERROR: 2 RID allocations of type 'N10RendererRD12LightStorage11ShadowAtlasE' were leaked at exit.
ERROR: 1 RID allocations of type 'N10RendererRD12LightStorage15ReflectionAtlasE' were leaked at exit.
ERROR: 1 RID allocations of type 'N10RendererRD14TextureStorage12RenderTargetE' were leaked at exit.
ERROR: 2 RID allocations of type 'N10RendererRD14TextureStorage7TextureE' were leaked at exit.
ERROR: 1 RID allocations of type 'N18RendererCanvasCull6CanvasE' were leaked at exit.
ERROR: 1 RID allocations of type 'N16RendererViewport8ViewportE' were leaked at exit.
ERROR: 1 RID allocations of type 'N17RendererSceneCull8ScenarioE' were leaked at exit.
WARNING: 4 RIDs of type "Texture" were leaked.
     at: finalize (drivers/vulkan/rendering_device_vulkan.cpp:9290)
WARNING: ObjectDB instances leaked at exit (run with --verbose for details).
     at: cleanup (core/object/object.cpp:1982)
```
I checked in 4.0rc1 (v4.0.rc1.mono.official [8843d9ad3]) and it has same result as beta 17.
I checked also in non mono version of Godot (v4.0.rc1.official [8843d9ad3] and v4.0.beta15.official [4fa6edc88]) on this more basic project and they act the same way as mono version.

### Steps to reproduce

1. Open project
2. Export Project to Windows
3. Try to run exported project

### Minimal reproduction project

I tested using this basic project: https://github.com/patrykjadamczyk/fluffy-spork

**Comment by RedworkDE** [comment]
Regression from #72031
Somehow the imported texture of the skybox is not included in the exported project; If I manually copy it into the exported zip it works.
Enabling all texture formats during export does not help.
The issue appears to be somewhat specific to this the texture, a few other random EXR images i found on my computer there are no problems.

The problem remap section of the import file is:
```
[remap]

importer="texture"
type="CompressedTexture2D"
uid="uid://bfcsdsu4el803"
path.bptc="res://.godot/imported/industrial_sunset_02_puresky_4k.exr-fb33adb88c5253d1c44585a4ba1f1a66.bptc.ctex"
metadata={
"imported_formats": ["s3tc_bptc"],
"vram_texture": true
}
```
while the others have
```
[remap]

importer="texture"
type="CompressedTexture2D"
uid="uid://b4ghb208datdv"
path="res://.godot/imported/Diffuse.exr-cbce9846473d02c1d293cdd9b974d14c.ctex"
metadata={
"vram_texture": false
}
```
That's probably related, but beyond that, IDK.

**Comment by akien-mga** [comment]
This might have been "fixed" by #73136, though you will need to delete the `.import` files to see the effect.

But since that PR only changed the default compression from BPTC to uncompressed, the bug would likely still happen if manually changing the compression. There must be something wrong in the export code for lightmaps.

**Comment by akien-mga** [comment]
Duplicate of #72856
